packages/e2e-tests/tests/backup-tests.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { expect, type Page } from '@playwright/test'
+
+import {
+  clickThroughTestIds,
+  createProfiles,
+  deleteAllProfiles,
+  getUser,
+  loadExistingProfiles,
+  reloadPage,
+  switchToProfile,
+  test,
+  User,
+} from '../playwright-helper'
+
+test.describe.configure({ mode: 'serial' })
+
+let existingProfiles: User[] = []
+const numberOfProfiles = 1
+let page: Page
+
+test.beforeAll(async ({ browser, isChatmail }) => {
+  const contextForProfileCreation = await browser.newContext()
+  const pageForProfileCreation = await contextForProfileCreation.newPage()
+
+  await reloadPage(pageForProfileCreation)
+
+  existingProfiles =
+    (await loadExistingProfiles(pageForProfileCreation)) ?? existingProfiles
+
+  await createProfiles(
+    numberOfProfiles,
+    existingProfiles,
+    pageForProfileCreation,
+    browser.browserType().name(),
+    isChatmail
+  )
+
+  await contextForProfileCreation.close()
+
+  page = await browser.newPage()
+  await reloadPage(page)
+})
+
+test.afterEach(async () => {
+  for (let i = 0; i < 5; i++) {
+    await page.keyboard.press('Escape')
+  }
+  existingProfiles = await loadExistingProfiles(page)
+})
+
+test.afterAll(async ({ browser }) => {
+  await page?.close()
+
+  const context = await browser.newContext()
+  const pageForProfileDeletion = await context.newPage()
+  await reloadPage(pageForProfileDeletion)
+  await deleteAllProfiles(pageForProfileDeletion, existingProfiles)
+  await context.close()
+})
+
+test('shows warning when scanning backups that are newer than supported', async ({
+  browserName,
+}) => {
+  if (browserName.toLowerCase().includes('chrom')) {
+    await page.context().grantPermissions(['clipboard-read', 'clipboard-write'])
+  }
+
+  const user = getUser(0, existingProfiles)
+  await switchToProfile(page, user.id)
+
+  await page.getByTestId('open-settings-button').click()
+
+  const settingsDialog = page.getByTestId('settings-dialog')
+  await expect(settingsDialog).toBeVisible()
+
+  await settingsDialog
+    .getByRole('button', { name: 'Add Second Device' })
+    .click()
+
+  const sendBackupDialog = page
+    .locator('dialog')
+    .filter({ hasText: 'Add Second Device' })
+  await expect(sendBackupDialog).toBeVisible()
+
+  await sendBackupDialog.getByRole('button', { name: 'Continue' }).click()
+
+  const copyButton = sendBackupDialog.getByRole('button', { name: 'Copy' })
+  await expect(copyButton).toBeVisible()
+  await copyButton.click()
```

Comment:
```suggestion
  await sendBackupDialog.getByRole('button', { name: 'Copy' }).click()
```

A successful `click()` implies that the button is visible.